src/blocks/after/index.html

```diff
@@ -0,0 +1,19 @@
+<html>
+    <head>
+    	 <meta name="viewport" content="width=device-width, initial-scale=1">
+          <title>Animation of dog,cat and rat</title>
+        <link rel="stylesheet" type="text/css" href="style.css">
+	</head>
+    <body background="bg.jpg">
```

Comment:
You need to commit the images to git, too, or this won't work. Or link to a public URL.